agent-inject/agent/annotations.go

```diff
@@ -600,6 +609,15 @@ func (a *Agent) templateConfigExitOnRetryFailure() (bool, error) {
 	return strconv.ParseBool(raw)
 }
 
+func (a *Agent) templateConfigStaticSecretRenderInterval() (string, error) {
+	raw, ok := a.Annotations[AnnotationTemplateConfigStaticSecretRenderInterval]
+	if !ok {
+		return DefaultTemplateConfigStaticSecretRenderInterval, nil
+	}
+
+	return raw, nil
+}
+
```

Comment:
This function is no longer needed if we're not setting a default as suggested above.
```suggestion
```